utils/format-error-for-apollo.js

```diff
@@ -0,0 +1,14 @@
+const formatError = err => {
+  const { originalError } = err;
+
+  if (originalError && originalError.name === 'RuleError') {
```

Comment:
Do we have errors constants, so we won't rely on magics strings?

If no I suggest to create one and it will look like
`const ERROR_TYPES = {
   'RULE_ERROR': 'RuleError'
}`